### ISSUE TITLE
Make changes from the selected profile always visible

### ISSUE THREAD

**Issue by dwery** [Ultimaker/Cura#9432]
### Is your feature request related to a problem?

I change a setting that is not currently visible and there is no indication in the UI

### Describe the solution you'd like

Have an option to make settings that are changes from the currently selected profile always visible

### Describe alternatives you've considered

n/a

### Affected users and/or printers

All

**Comment by fvrmr** [comment]
Hi @dwery thank you for your feature request. 
If you will click on recommend you will see an arrow that will show changes you have made. 

For example:
I have my settings visibility on basic and I change the `line width` from 0.35mm to 0.8mm by searching for line.
![Screenshot (133)](https://user-images.githubusercontent.com/77000924/112134386-fd0cdd00-8bcc-11eb-9d52-bdb799af9ca8.png)


Then I go to recommend 

![Screenshot (132)](https://user-images.githubusercontent.com/77000924/112135488-20845780-8bce-11eb-9df2-4c187b0aa77b.png)


And you will see a arrow and if you hover over the arrow this message comes up:

![Screenshot (135)](https://user-images.githubusercontent.com/77000924/112135565-3265fa80-8bce-11eb-8410-e5aaa8053b31.png)

Please let me know if this works for you.

**Comment by dwery** [comment]
Hi, this helps noticing it, but in order to know what changed you would still have to go back to custom, expand everything and check for the mark. It would be much easier to just make changed settings always visible.

**Comment by yet-another-average-joe** [comment]
Aren't recommended settings for Ultimaker printers only, with listed filaments ?

This being said, an option in the filter menu ("Modified settings", relative to the current profile) would be very usefull.

**Comment by Ghostkeeper** [comment]
> Aren't recommended settings for Ultimaker printers only, with listed filaments ?

No, the "recommended" settings are simply the default profiles without any individual modifications. Every dot is one of the profiles that would be available from the quality profile drop-down. These profiles are created by contributors in most cases (although some printers use the default "global" set of profiles, like Custom FFF Printer, which was originally based on the Ultimaker 2 profiles).

We've had an issue in our backlog for a long time to implement a setting visibility pre-set option for "all currently changed settings". Developers, refer to issue CURA-5421. Our product managers also seem to find this useful (and I agree too) but while it's on our priority list, it's still about halfway down the list right now.
